docs/provision_documentation.sh

```diff
@@ -0,0 +1,2 @@
+docker build -t  wazuh-qa-documentation .
```

Comment:
```suggestion
#!/usr/bin/env bash

docker build -t  wazuh-qa-documentation .
```

Reply:
Done in 618a8892f93dd794a6000baa838503371ec1eb6d